minimint/Cargo.toml

```diff
@@ -41,6 +41,4 @@ tokio = { version = "1.0.1", features = ["full"] }
 tokio-rustls = "0.23.4"
 tokio-util = { version = "0.6.0", features = [ "codec" ] }
 tracing ="0.1.22"
-tracing-opentelemetry = "0.17.2"
```

Comment:
Are you sure about that one? Feels like it should be optional but not entirely removed. @Maan2003 what do you think?

Reply:
yes this should be optional, enabled by `telemetry` feature